src/plugins/privilege/privilege.go

```diff
@@ -144,6 +144,8 @@ func (p *Privilege) Check(database string, user string, node sqlparser.Statement
 	if node != nil {
 		sqlparser.Walk(func(nod sqlparser.SQLNode) (kontinue bool, err error) {
 			switch nod := nod.(type) {
+			case *sqlparser.StarExpr:
```

Comment:
Here ColName has TableName too, we also need to skip out:
https://github.com/radondb/radon/blob/master/src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/ast.go#L1866

If not, this queryr will be failed(no need to check TableName in column):
```select a.c1 from test.t1 as a```

Reply:
Yes, ColName, 
should check all usages of the TableName.